resources/lang/de/Debug.properties

```diff
@@ -1,2 +1,4 @@
 filterResponse=Filter Antwort
 FeedbackService_itemDoesNotExistError=Der angeforderte Artikel existiert nicht
+determineVisibilityResponse = Determine visibility response
```

Comment:
Where is this used?

Reply:
Was used for log debugging purposes. I've removed it now.